### ISSUE TITLE
Unable to change item's id with just keyboard 

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#2257]
currently all of ids values are auto-generated on creation and UI components is disabled by default: 

![image](https://github.com/NationalSecurityAgency/skills-service/assets/51749745/7414995a-5d5d-4306-a8fd-32f206ed6ccb)

you can easily tab to the Enable link however pressing `Enter` closes the modal

Preferred solution would be to change the Enable control to be a switch using `b-form-checkbox` with `switch` option; this way spacebar can be used to enable the id field
